README.md

```diff
@@ -5,6 +5,7 @@ markdown content. Includes:
 
 * [helm](https://github.com/helm/helm)
 * [helm-docs](https://github.com/norwoodj/helm-docs)
+* [helm-schema](https://github.com/ubergesundheit/helm-schema/tree/no-additional-properties-and-multi-type) (fork)
```

Comment:
Oh wait, this isn't correct any more right?

Reply:
Might also be worth adding an extra example into the `## Usage` section below about how to run the helm-schema directly (if there's anything special you need to do to make it work right, might be the same as the helm-docs example?)

Reply:
oh yes, you're right! :fearful:  I'll update it!

Reply:
fixed in https://github.com/cilium/helm-toolbox/pull/3/commits/13fa18cdbc760981ae3396dc081850bf8cf79691